plugins/inputs/postgresql/postgresql.go

```diff
@@ -42,6 +40,20 @@ var sampleConfig = `
   ##
   address = "host=localhost user=postgres sslmode=disable"
 
+  ## connection configuration.
+
+  ## maxidle - specify the number of idle connections to maintain. see sql.DB for details.
+  ## default is 1. setting to 0 to force disconnect after use.
+  max_idle = 1
+
+  ## maxopen - specify the maximum number of connections to maintain. see sql.DB for details.
+  ## default is 1.
+  max_open = 1
```

Comment:
also I am unsure of telegraf's collection behaviour if a input takes longer than the collection period. Does it invoke Gather again? if it does then this setting has a potential purpose. otherwise I am in favor of leaving it out of the documentation.